pages/common/dfc.md

```diff
@@ -0,0 +1,20 @@
+# dfc
+
+> Gives an overview of the file system disk space usage with colours and graphs.
+> See also `df`.
```

Comment:
Maybe we can put the link to the wiki page:
```suggestion
> More information: <https://projects.gw-computing.net/projects/dfc/wiki>.
```

Reply:
Probably better than just referring to df by name so I'll make that change